apps/meteor/client/views/room/modals/E2EEModals/EnableE2EEModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import GenericModal from '../../../../components/GenericModal';
+
+type EnableE2EEModalProps = {
+	onConfirm: () => void;
+	onClose: () => void;
+	roomType: string;
+};
+
+const EnableE2EEModal = ({ onConfirm, onClose, roomType }: EnableE2EEModalProps): ReactElement => {
+	const { t } = useTranslation();
+
+	return (
+		<GenericModal
+			icon='key'
+			title={t('E2E_enable_encryption')}
+			variant='warning'
+			confirmText={t('E2E_enable_encryption')}
+			onConfirm={onConfirm}
+			onCancel={onClose}
+		>
+			<Box mbe={16} is='p'>
+				<Trans i18nKey='E2E_enable_encryption_description' tOptions={{ roomType }} />
```

Comment:
![image](https://github.com/user-attachments/assets/fe55bb60-bde3-47d7-9b18-3c79d8c47fe8)
This button is not getting the translation right, `roomType` is not being replaced